index.html

```diff
@@ -55,6 +55,189 @@ <h2 class="headline-text">
+                </li>
+                <li>
+                    <button class="lang-button">HTML</button>
+                </li>
+                <li>
+                    <button class="lang-button">Bootstrap</button>
+                </li>
+                <li>
+                    <button class="lang-button">Ruby</button>
+                </li>
+            </ul>
+            <button class="see-project1">See Project</button>
+        </div>
+        <div class="pages-container">
+            <div class="page">
+                <h4 class="page-header">Profesional Art Printing Data</h4>
+                <p class="page-paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="page-language">
+                    <li>
+                        <button class="page-lang-button">HTML</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Bootstrap</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Ruby</button>
+                    </li>
+                </ul>
+                <button class="page-see-project">See Project</button>
+            </div>
+            <div class="page">
+                <h4 class="page-header">Profesional Art Printing Data</h4>
+                <p class="page-paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="page-language">
+                    <li>
+                        <button class="page-lang-button">HTML</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Bootstrap</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Ruby</button>
+                    </li>
+                </ul>
+                <button class="page-see-project">See Project</button>
+            </div>
+            <div class="page">
+                <h4 class="page-header">Profesional Art Printing Data</h4>
+                <p class="page-paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="page-language">
+                    <li>
+                        <button class="page-lang-button">HTML</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Bootstrap</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Ruby</button>
+                    </li>
+                </ul>
```

Comment:
[OPTIONAL]
- [ ] Kindly you may consider removing your tech stacks in the `button` and using only the generic `li` since the `button` don't have any semantic meaning here.